src/components/ClassModulesProgressCard/ClassModulesProgressCard.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import Card from 'components/Card/Card';
+import { CircularProgressbarWithChildren } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import Carousel from 'components/Carousel/Carousel';
+
+import { ModuleList, getDummyData } from './dummyData';
+import styles from './ClassModulesProgressCard.module.css';
+
+type Props = {
+  className?: string;
+};
+
+export default function ClassModulesProgressCard({ className }: Props) {
+  const [moduleList, setModuleList] = useState<ModuleList>([]);
+
+  const moduleNodes = moduleList.map((module, index) => (
+    <div
+      key={module.id}
+      className={styles['progress-bar-container']}
+    >
+      <div className={styles['progress-bar']}>
+        <CircularProgressbarWithChildren value={module.percentage}>
```

Comment:
Was this easy to use? I did a component to homogenize all CircularProgressbar, but if this works don't waste time changing it
Just as a note for the future that the other component is also available